tests/Backend/NetworkPolicies/SnowflakeNetworkPoliciesTest.php

```diff
@@ -85,14 +90,20 @@ public function testAccessWithSystemNetworkPolicy(): void
         $this->dropNetworkRule($testNetworkRuleName);
     }
 
+    // this test won't be working locally because IP of local connection and IP of test runner are the same
```

Comment:
Proč by tenhle test nefungoval lokálně?

IPs jsou stejná, ale u účtu, co se používá k přímému připojení na DB a k manipulaci s NP není NP přiřazena, takže jej neomezí a je omezen jen projektový účet.

Reply:
protoze existuje jen jedna network policy a IP connection a IP SAPI runneru je lokalne stejna (na CI ne). A ty potrebujes, aby projektovy uzivatel se na SNFLK dobouchal a udelal WS a vsechno co connection ma udelat, proto se ocekava, ze `$systemNetworkPolicyName` ma rule odpovidajici IP connection.

A pak na radku 130 se snazis dobouchat do WS, ktery ma taktez na sobe NP `$systemNetworkPolicyName`, ve ktere se v tu chvili skryva IP connection a `10.0.0.1`, ale ani jedna z nich v CI a v realnem usecase neodpovida IP exekutora testu (aka SAPI runneru). Problem je v tom, ze lokalne to odpovida. Proto ten test lokalne pada

Reply:
pred tim to nevadilo, protoze nebylo potreba aby projekt mel nastavenou NP (ted to potreba je) a mohl v ni byt tedy nejakej bordel . 

to co bych s tim mohl udelat je to nekde na lajne 128 procistit - z nasi projektove NP vyhazet vsechny ruly s vyjimkou te `10.0.0.1`

Reply:
jo, klidne, udelam task, chce to vic promyslet, aby ten test nebyl krehky